pkg/credentialprovider/plugin/plugin.go

```diff
@@ -445,8 +445,13 @@ func (e *execPlugin) runPlugin(ctx context.Context, cmd *exec.Cmd, image string)
 		return fmt.Errorf("error execing credential provider plugin %s for image %s: %w", e.name, image, ctx.Err())
 	}
 	if err != nil {
+		stderr := ""
+		if err2, ok := err.(*exec.ExitError); ok {
```

Comment:
naive question, is there any possibility of leaking sensitive information?

Reply:
That's a fair question. However, if the credential providers are logging sensitive information, that's where the bug is. They shouldn't be logging creds or anything.

Reply:
fair enough